src/shared/DeleteModal/DeleteModal.test.tsx

```diff
@@ -17,10 +17,8 @@ describe("DeleteModal", () => {
     const onDelete = jest.fn();
     const onCancel = jest.fn();
 
-    const comp = render(
-      <div id={"root"}>
```

Comment:
I didn't want to pollute tests with these wrapping divs anymore to test modals. So I removed them and added the `renderDialog` helper to standardize this.